app/pages/lab/workflows-container.jsx

```diff
@@ -66,6 +66,7 @@ export default class WorkflowsContainer extends React.Component {
     workflow.update({ active: checked }).save()
       .catch(error => console.log(error))
       .then(() => {
+        this.props.project.uncache();
```

Comment:
Does uncaching the project here cause any problems updating the resource in the next couple of lines? I don't think it will but just checking.

Reply:
No, as far as I am aware.

Reply:
👍